app/models/preorder_information.rb

```diff
@@ -41,15 +41,15 @@ def infer_status
     elsif school_will_order_devices? && any_school_users? && chromebook_information_complete?
       if school.can_order_devices_right_now?
         'school_can_order'
-      elsif (school.std_device_allocation&.devices_ordered || 0).positive?
+      elsif (school.std_device_allocation&.devices_ordered || 0).positive? # bug should do all device types
```

Comment:
Good spot! Please could you add a Trello card if you don't want to fix it in this PR?

Reply:
I've raised a ticket for this to keep track of it